mrblib/itamae/resource_executor/user.rb

```diff
@@ -0,0 +1,82 @@
+module Itamae
+  module ResourceExecutor
+    class User < Base
+      def apply(current, desired)
+        if desired.exist
+          if exist?(desired.username)
```

Comment:
Thank you for keeping codebase. While this code is acceptable, it'd be better to use `current.exist` here.